metron-interface/metron-config/src/app/model/threat-triage-config.ts

```diff
@@ -0,0 +1,23 @@
+import {RiskLevelRule} from './risk-level-rule';
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export class ThreatTriageConfig {
+  riskLevelRules: RiskLevelRule[] = [];
+  aggregator: string = 'MAX';
```

Comment:
I disagree.  There is already an enum for this (org.apache.metron.common.aggregator.Aggregators) so a REST call should be added to populate this on the angular side.